web/server/www/scripts/login/login.js

```diff
@@ -75,8 +75,10 @@ function (declare, cookie, dom, domConstruct, domClass, ioQuery, keys, on,
       this.set('isAlreadyLoggingIn', true);
       this._standBy.show();
 
+      var userName = this.txtUser.get('value');
```

Comment:
In line 69 there is also a place where the value of the text box should be queried by `get()` function.

Reply:
Good catch. I fixed it.